UnityProject/Assets/Scripts/Objects/GasContainer.cs

```diff
@@ -68,7 +68,8 @@ public GasMix GasMix
 		[Tooltip("If true : Cargo will accept gases found within this container and can be sold.")]
 		public bool CargoSealApproved = false;
 
-		[SerializeField] private bool exploadOnTooMuchDamage = true;
+		[FormerlySerializedAs("exploadOnTooMuchDamage")] [SerializeField]
```

Comment:
remove the formerlyserialized as. Re do your modification on the closet prefab instead.